asm_utils/src/reachability.rs

```diff
@@ -168,25 +168,32 @@ pub fn extract_label_offsets<R: Register, F: FunctionOpKind>(
         })
 }
 
-pub fn references_in_statement<R: Register, F: FunctionOpKind>(
+pub fn references_in_statement<R: Register, F: FunctionOpKind, A: Architecture>(
```

Comment:
Not in this PR, but later: It might make sense to have Register and FunctionOpKind as a trait-type of Architecture, if that works.

Reply:
Either write an issue, or we will forget.